src/components/heading/heading.spec.js

```diff
@@ -252,7 +281,7 @@ describe("Heading", () => {
         />
       );
 
-      elementsTagTest(wrapper, ["back", "help", "subtitle", "title", "pills"]);
+      elementsTagTest(wrapper, ["help", "subtitle", "title", "pills"]);
```

Comment:
I'm probably missing something but I don't see why you've removed "back" from the `data-element` properties we're testing. It looks to me like that should still be there after your changes? If the test fails for some reason with that in I'd rather understand why and fix it up, than just remove it.

Reply:
I'm already using that tag on line 235 to find the button, so looking for it again seems redundant